prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -82,7 +82,7 @@ class ReportFunction {
                     context.logger.info("Successfully reported: ${validatedRequest.report.id}.")
                     val destinations = mutableListOf<String>()
                     routeReport(context, workflowEngine, validatedRequest, destinations, actionHistory)
-                    val responseBody = createResponseBody(validatedRequest, destinations)
+                    val responseBody = createResponseBody(validatedRequest, destinations, actionHistory)
                     workflowEngine.receiveReport(validatedRequest.report)
```

Comment:
Future refactor would change this to something simpler like
`val responseBody = actionHistory.createResponseBody()`